### ISSUE TITLE
Enhancement: Bulk operations for Postgresql

### ISSUE THREAD

**Issue by stefandevo** [mikependon/RepoDb#660]
We are considering moving to Postgresql but we love the Bulk operations of repoDb. 
Any chance that Postgresql will get support?
Note sure if https://github.com/neisbut/Npgsql.Bulk could help for example?

**Comment by mikependon** [comment]
Yeah, and also to MySQL. We will start this on January 2021. :)

**Comment by mikependon** [comment]
FYI: In order for me to clear the possible technical debt and focus on this feature, I have been working to finalize and maturize the latest version of the library at v1.12.5, as reported by some users.

But this one is next in the line, so please wait for this feature soon!

Also, please feel free to share your thoughts if you have some (specifically the experience) on how this feature is to be implemented.

**Comment by mikependon** [comment]
The first quarter is a bit challenging given with the timelines, so applogy for that. Can you wait for this to the max 2nd quarter of 2021? There were recent PRs and bugs requested that needs to be fixed ASAP.

Personally, I am bit occupied due to some talks preparation as well. But, I would like to architect how this thing will happen for PostgreSQL as I initially thought.

Initial design could be Feb but the actuals would be 2nd quarter around April or May. Hope that is okay?

**Comment by stefandevo** [comment]
Of course. It is a free product so I respect your personal agenda! If I need it sooner I will look for some alternatives. I can switch back later.

**Comment by stefandevo** [comment]
@mikependon just checking the status. Thx.

**Comment by stefandevo** [comment]
Hi @mikependon just a quick question for this; is this somewhere on your roadmap or should I move on finding other solutions? Just let me know so I can proceed. Thx.

**Comment by mikependon** [comment]
@stefandevo - yes, it is on our roadmap, in fact, we have started the analysis and the architecture design of this and even the initial project (few months back). However, everything we have worked pertaining to this is not yet pushed to the repository, as we were prioritizing the bugs and critical reports during the last improvements.

In relation to the timeline, we are planning to push one more major release and will start revisiting the PosgtreSQL Bulk Operations solution. The major release will come in the max of 2 months from now (as of writing this). I may say that the delivery of this will be on the last quarter. I personally hope that we are not due yet based on your needs, but if you're on hurry, I suggest, write your own Bulk Operations for now up until we deliver this.

Hoping for your patience here mate. 🌱

**Comment by mikependon** [comment]
Referencing #372.

**Comment by mikependon** [comment]
Typo: All references should be for #372, but is not for #327.
